.github/workflows/build_test.yml

```diff
@@ -0,0 +1,94 @@
+name: Build and Test
+
+on:
+  push:
+    branches: [master]
```

Comment:
Isn't is a PR check ?

Reply:
correct. use only on: pull_request

Reply:
Yes it is based on pull request and on push to master. I like the idea that there is visible status of build/test after push to master, we can put it into a readme. wdyt?

Reply:
build and test on master are already performed on Jenkins. this is extra work and status results are not very useful in public as we publish snapshots/stables regularly.

Reply:
done